src/System.Private.ServiceModel/src/System/ServiceModel/Channels/MessageContent.cs

```diff
@@ -167,7 +167,7 @@ internal static HttpContent Create(HttpChannelFactory<IRequestChannel> factory,
 
     public class StreamedMessageContent : MessageContent
     {
-        private Task _encodingTask;
+        private const int WriteBufferSize = 4096;
```

Comment:
Good point. It was because HttpWebRequest used 4K internally for it's own buffer so I just copied that.